src/ctapipe/containers.py

```diff
@@ -1132,40 +1111,172 @@ class TelescopePointingContainer(Container):
     altitude = Field(nan * u.rad, "Altitude", unit=u.rad)
 
 
-class PointingContainer(Container):
-    tel = Field(
-        default_factory=partial(Map, TelescopePointingContainer),
-        description="Telescope pointing positions",
-    )
+class ArrayPointingContainer(Container):
+    """
+    Container holding pointing information for the combined array
+    after all necessary correction and calibration steps.
+    """
+
     array_azimuth = Field(nan * u.rad, "Array pointing azimuth", unit=u.rad)
     array_altitude = Field(nan * u.rad, "Array pointing altitude", unit=u.rad)
     array_ra = Field(nan * u.rad, "Array pointing right ascension", unit=u.rad)
     array_dec = Field(nan * u.rad, "Array pointing declination", unit=u.rad)
 
 
-class EventCameraCalibrationContainer(Container):
+# Camera containers
+class CameraCalibrationContainer(Container):
     """
-    Container for the calibration coefficients for the current event and camera
+    Storage of camera calibration coefficients for a given time.
     """
 
-    dl1 = Field(
-        default_factory=DL1CameraCalibrationContainer,
-        description="Container for DL1 calibration coefficients",
+    time = Field(
+        NAN_TIME,
+        "validity time of the camera calibration coefficients.",
+    )
+    pedestal_offset = Field(
+        None,
+        "Residual mean pedestal of the waveforms for each pixel."
+        " This value is subtracted from the waveforms of each pixel before"
+        " the pulse extraction. Shape: (n_channels, n_pixels)",
     )
+    factor = Field(
+        None,
+        "Multiplicative coefficients for the calibration of extracted charge"
+        " into physical units (e.g. photoelectrons or photons) for each pixel."
+        " The coefficients include the relative correction between pixels to"
+        " achieve a uniform charge response. Shape: (n_channels, n_pixels)",
+    )
+    time_shift = Field(
+        None,
+        "Additive coefficients for the timing correction before charge extraction"
+        " for each pixel. Shape: (n_channels, n_pixels)",
+    )
+    outlier_mask = Field(
+        None,
+        "Boolean mask indicating which pixels are considered outliers."
+        " Shape: (n_channels, n_pixels)",
+    )
+    is_valid = Field(
+        False,
+        (
+            "True if the coefficients are valid, False if they are not valid or "
+            "if a high fraction of faulty pixels exceeding the pre-defined threshold "
+            "is detected during the time period."
+        ),
+    )
+
 
+class StatisticsContainer(Container):
+    """Store descriptive statistics of a pixel-wise quantity for each channel"""
 
-class EventCalibrationContainer(Container):
+    mean = Field(
```

Comment:
Why is the number of entries used to compute the stats gone here? This is important information.

Reply:
maybe we can move `is_valid` and `n_events` to the `StatisticsContainer`?